ngraph_bridge/ngraph_builder.cc

```diff
@@ -4825,6 +4825,55 @@ static Status TranslateUnpackOp(const Node* op,
   return Status::OK();
 }
 
+static Status TranslateUnsortedSegmentSumOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  /*
+  # Code from python that reflects how unsorted_segment_sum
+  # works in tensorflow using scatter_add
+
+  def unsorted_segment_sum(inputs, segment_ids, num_segments):
+    inputs = tf.convert_to_tensor(inputs)
+    segment_ids = tf.convert_to_tensor(segment_ids)
+
+    shape = inputs.get_shape().as_list()
+    seg_ids = segment_ids.get_shape().as_list()
+    result = tf.Variable(tf.zeros([num_segments] + shape[len(seg_ids):],
+                                  dtype=inputs.dtype))
+
+    return tf.scatter_add(result, segment_ids, inputs)
```

Comment:
lets add this as a pytest.
Examples can be found here: https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/test_prod.py

Reply:
I don't see why this should be a pytest? The comment was there only to show how we can replace `UnsortedSegmentSum` with `ScatterAdd`.